rpcs3/Emu/RSX/GL/GLGSRender.cpp

```diff
@@ -723,6 +723,7 @@ void GLGSRender::load_program_env()
 
 	const u32 handled_flags = (rsx::pipeline_state::fragment_state_dirty | rsx::pipeline_state::vertex_state_dirty | rsx::pipeline_state::transform_constants_dirty | rsx::pipeline_state::fragment_constants_dirty | rsx::pipeline_state::fragment_texture_state_dirty);
 	m_graphics_state &= ~handled_flags;
+	vertex_constants_dirty = {};
```

Comment:
Drop this, it's not meaningful to leave in dead code unless a major breakthrough is achieved.

Reply:
Done.